project_task_pull_request_state/models/project_task.py

```diff
@@ -0,0 +1,41 @@
+# Copyright Cetmix OU 2023
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0).
+from odoo import api, fields, models
+
+
+class ProjectTaskState(models.Model):
```

Comment:
```suggestion
class ProjectTask(models.Model):
```
It would be better to follow the common naming as in https://github.com/OCA/project/pull/1189/files#r1440500916

Reply:
@leemannd, thank you for suggestion. I added this to the last commit.